index.Rmd

```diff
@@ -4,7 +4,7 @@ documentclass: book
 site: bookdown::bookdown_site
 bibliography: [book.bib]
 biblio-style: apalike
-link-citations: yes
+link-citations: no
```

Comment:
Might be justified but what is the justification? Is it better to have no than yes here?

Reply:
It's because of the issue **Bibliography in the end? #101**. This removes from individual pages but if it is needed we can leave it there.

Reply:
Ok great, now I understand! This is good if it works.